composer.json

```diff
@@ -26,22 +26,24 @@
         "php": ">=5.3.6",
         "ext-zip": "*",
         "ext-fileinfo": "*",
-        "symfony/console": "~2.3",
-        "symfony/finder": "~2.2",
-        "symfony/filesystem": "~2.1",
-        "symfony/process": "~2.1",
+        "symfony/console": ">=2.5||~3.0",
```

Comment:
Agreed. Should be "~2.3||~3.0" (or "~2.3|~3.0", I still need to understand the question about single or double pipe)